extensions/amp-analytics/0.1/vendors/comscore.js

```diff
@@ -26,6 +26,7 @@ const COMSCORE_CONFIG = jsonLiteral({
     'pageview':
       '${base}c1=2' +
       '&c2=${c2}' +
+      '&cs_ucfr=$IF($EQUALS(${consentState}, sufficient), 1)$IF($EQUALS(${consentState}, insufficient), 0)$IF($EQUALS(${consentState}, ), )' +
```

Comment:
I don't think the nested macros will currently work like this, have you tested?

Reply:
Yeah, I tested it I can send you my test page if you'd like to see yourself

Reply:
Great! I believe you :) thanks for confirming.